backend/runners/load_representative_points.py

```diff
@@ -228,4 +270,10 @@ def _sanitize_county_name_in_service_area_id(county, replacement_mapping=CHARACT
 
 
 if __name__ == '__main__':
-    _main(_get_arguments())
+    arguments = _get_arguments()
+    print('Adding file - %s' % arguments['filepath'])
+    try:
+        _main(arguments)
+    except Exception as e:
+        print('An error occured uploading the file.')
```

Comment:
There are already `try...excepts...` in the individual inserts for service areas and RPs, so this won't get run unless the error is in transforming the features.